docs/userguide/modules/getting-started/partials/creating-db-postgresql.adoc

```diff
@@ -0,0 +1,85 @@
+== Creating a PostgreSQL database instance
+
+For the application to use a PostgreSQL database service, you must create a PostgreSQL database instance.
+
+To create a PostgreSQL database instance, you must create the following Kubernetes resources:
+
++++ <details><summary> +++
+`Deployment` resource to run the actual database instance
++++ </summary><div> +++
+[source,yaml]
+----
+include::attachment$postgresql-deployment.yaml[tag=db-deployment]
+----
++++ </div></details> +++
+
++++ <details><summary> +++
+`Secret` resource to store the database credentials
++++ </summary><div> +++
+[source,yaml]
+----
+include::attachment$postgresql-deployment.yaml[tag=db-secret]
+----
++++ </div></details> +++
+
++++ <details><summary> +++
+`Service` resource to provide a way to access to the database
++++ </summary><div> +++
+[source,yaml]
+----
+include::attachment$postgresql-deployment.yaml[tag=db-service]
+----
++++ </div></details> +++
+
+[discrete]
+=== Procedure
+
+. Create the database resources by running the following command:
++
+[source,bash,subs="attributes,macros"]
+----
+kubectl apply -f link:{attachmentsdir}/postgresql-deployment.yaml[pass:a[{site-url}/{page-component-name}/{page-module}/{attachmentsdir}/postgresql-deployment.yaml]] -n {quickstart-namespace}
+----
++
+. After you have created the database instance, verify that the respective pod in the `{quickstart-namespace}` namespace
+is up and running (it will take less than a minute):
++
+[source,bash,subs="attributes"]
+----
+kubectl get pods -n {quickstart-namespace}
+----
++
+.Example output:
++
+----
+NAME                                          READY  STATUS   RESTARTS  AGE
+spring-petclinic-postgresql-6db5594876-4556g  1/1    Running  0         16m
+----
++
+The previous output verifies that the setup of the database for the application is complete.
+You can deploy the sample application and connect it to the database service.
+
+[IMPORTANT]
+====
+The `Deployment` resource of the database has a couple of annotations set in the `.metadata.annotations` section:
```

Comment:
```suggestion
The `Deployment` resource of the database has few annotations set in the `.metadata.annotations` section:
```

Reply:
It is actually: The `Deployment` resource of the database has a few annotations set in the `.metadata.annotations` section:

It is a minor comment. Considering that this is the only comment that is pending and I am also working on a PR for the upstream content, if it helps, I can make this change along with my work. But this will take some time as I am working on the QSG content. I hope that is ok. @pmacik I may need your help if I run into errors during acceptance tests.